build.gradle.kts

```diff
@@ -9,6 +9,7 @@ plugins {
   id("org.openapi.generator") version "7.13.0"
   kotlin("plugin.spring") version "2.1.21"
   kotlin("plugin.jpa") version "2.1.21"
+  id("dev.zacsweers.redacted") version "1.13.0"
```

Comment:
Notice this new addition please.

This is a safeguard accidental logging of fields containing sensitive data.

One of the things with data classes is you get a default implementation of toString() which is easy to miss.

This library allows you to annotate fields you want redacted from data classes when toString() is called e.g. via logging.